specs/merge/beacon-chain.md

```diff
@@ -230,19 +238,31 @@ def compute_timestamp_at_slot(state: BeaconState, slot: Slot) -> uint64:
 The implementation-dependent `ExecutionEngine` protocol encapsulates the execution sub-system logic via:
 
 * a state object `self.execution_state` of type `ExecutionState`
-* a state transition function `self.on_payload` which mutates `self.execution_state`
+* a state transition function `self.execute_payload` which applies changes to the `self.execution_state`
+* a method `self.consensus_validated` which notifies that the block holding the execution payload
+is valid with respect to the consensus rule set
 
-#### `on_payload`
+#### `execute_payload`
 
 ```python
-def on_payload(self: ExecutionEngine, execution_payload: ExecutionPayload) -> bool:
+def execute_payload(self: ExecutionEngine, execution_payload: ExecutionPayload) -> bool:
     """
     Returns ``True`` iff ``execution_payload`` is valid with respect to ``self.execution_state``.
     """
     ...
 ```
 
-The above function is accessed through the `EXECUTION_ENGINE` module which instantiates the `ExecutionEngine` protocol.
+#### `consensus_validated`
+
+```python
+def consensus_validated(self: ExecutionEngine, execution_payload: ExecutionPayload) -> None:
+    ...
+```
+
+The above functions are accessed through the `EXECUTION_ENGINE` module which instantiates the `ExecutionEngine` protocol.
```

Comment:
Suggest moving these notes to the `### Execution Engine` subsection prior to these function specific subsections.  (that is, right after the bullet points)

```
*Note*: `execute_payload` and `consensus_validated` are functions...

The body...
The Engine API...
```